docs/guided-tour/blueprints/external-blueprint/README.md

```diff
@@ -93,16 +108,15 @@ blueprint:
 
 - The field `context` contains the name of a custom resource of kind [Context](../../../usage/Context.md) in the same
   namespace as the Installation on the Landscaper resource cluster. [Our Context resource](./installation/context.yaml)
-  contains the information in which registry the component descriptor and blueprint are stored.
+  contains the information in which registry the component version with the component descriptor is stored. 
```

Comment:
Here it would also nice to have an inline example of a context and how the repository context is specified.

Reply:
Is on our list.